src/addons/par/icc/tools/icc_common.sh

```diff
@@ -145,6 +145,20 @@ then
     exit 1
 fi
 
+# Copy over any resource files if needed.
+resources_icc_path="$($get_config $config_db -n "" par.icc.resources_icc_path)"
+resources_tech_path="$($get_config $config_db -n "" par.icc.resources_tech_path)"
+resources_project_path="$($get_config $config_db -n "" par.icc.resources_project_path)"
+if [[ ! -z "$resources_icc_path" ]]; then
+    cp -r "${resources_icc_path}"/* $run_dir
```

Comment:
perhaps we should use -a to keep the other attributes of the file

Reply:
Done